src/global.scss

```diff
@@ -37,7 +37,7 @@
 
 /* Styles for 3rd prty libraries */
 @import "~nouislider/distribute/nouislider.min.css";
-@import "~swiper/scss";
+@import "~swiper/swiper.min.css";
```

Comment:
Minor import update as mentioned @jfmcquade  - I can't see any difference on the carousel component which uses swiper

Reply:
Yep, seems to work correctly